docs/Coiote_DM_Device_Onboarding/Device_Provisioning.md

```diff
@@ -0,0 +1,146 @@
+# Automated Provisioning for Nordic boards
+
+## Introduction
+With Factory provisioning for Nordic IoT devices, you can load on-device communication credentials and any cloud-related configuration at the factory level to automate secure device onboarding to Coiote IoT DM cloud.
+
+Here’s a tutorial to get you started with device provisioning using a dedicated script to be found in the [Anjay Zephyr Client repository](https://github.com/AVSystem/Anjay-zephyr-client).
+
+## Prerequisites
+- A Nordic board connected to your computer. 
+- Installed [Go Programming language](https://go.dev/dl).
```

Comment:
`export PATH=$PATH:home/<USER>/go/bin` might be needed, to be verified with some fresh system and installation step by step...

Reply:
checked on a fresh system, copy-pasting commands install mcumgr in `/home/<USER>/go/bin` and does not suggest adding it to `PATH`

Reply:
Added note about this.